server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -16,183 +16,185 @@ def get(self, section: str, option: str):
 
 
 class TestProcessTb:
-    @pytest.fixture(autouse=True)
-    def logger(self, test_logger):
-        # Setup the configuration and logger
-        self.logger = test_logger
-
-    def test_get_receive_dir_failed(self, monkeypatch):
+    def test_get_receive_dir_failed(self, monkeypatch, test_logger):
         def mock_get(section: str, option: str):
             if option == "pbench-receive-dir-prefix":
                 return ""
-            assert False
+            assert (
+                False
+            ), f"Unexpected configuration option, {section!r}/{option!r}, in mocked config.get()"
 
         expected_error_msg = "Failed: No value for config option pbench-receive-dir-prefix in section pbench-server"
         monkeypatch.setattr(MockConfig, "get", mock_get)
         with pytest.raises(ValueError) as e:
-            ProcessTb(MockConfig, self.logger)
+            ProcessTb(MockConfig, test_logger)
 
         assert expected_error_msg in str(e)
 
-    def test_get_receive_dir_value_failed(self, monkeypatch):
+    def test_get_receive_dir_value_failed(self, monkeypatch, test_logger):
         def mock_get(section: str, option: str):
             if option == "pbench-receive-dir-prefix":
-                return receive_dir / "wrong-directory"
-            assert False
+                return fixture_path / "wrong-directory"
+            assert (
+                False
+            ), f"Unexpected value for option, {section!r}/{option!r}, in mocked config.get()"
 
-        expected_error_path = str(receive_dir / "wrong-directory-002")
+        expected_error_path = str(fixture_path / "wrong-directory-002")
         monkeypatch.setattr(MockConfig, "get", mock_get)
         with pytest.raises(Exception) as e:
-            ProcessTb(MockConfig, self.logger)
+            ProcessTb(MockConfig, test_logger)
 
         assert expected_error_path in str(e)
```

Comment:
Aren't we expecting a specific exception (like `NotADirectoryError`)?  If so, we should probably be having PyTest catch it.

And, we probably want to find something more than just the path in the exception message text (like "is not a directory").